src/javascripts/components/Chat.js

```diff
@@ -1,19 +1,107 @@
-var React = require('react');
-var ChatBox_settings = require('./ChatBox_settings');
-var ChatBox = require('./ChatBox');
-var ChatInput = require('./ChatInput');
-
+import React from 'react';
+import ChatBoxSettings from './ChatBoxSettings';
+import ChatBox from './ChatBox';
+import ChatInput from './ChatInput';
 
 class Chat extends React.Component {
+  constructor() {
+      super()
+      this.state = {
+        open: true,
+        messages: [
+          {
+            user: "blue",
+            message: "blah blah blah"
+          }, 
+          {
+            user: "yellow",
+            message: "Lorem ipsum dipsum"
+          },
+          {
+            user: "yellow",
+            message: "Hallo lorem"
+          }
+        ]
+      }
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    const chatbox = document.querySelector('.chatbox')
```

Comment:
with React, it's better to grab things as a ref instead of with a query selector.  Of course, that means you'd need to move this code into the `ChatBox` component, but you can move this method there as-is and it'll still work. :)

Move it first and then replace the `const chatbox = document.querySelector('.chatbox')` with a ref